MFVNeutralino/test/TrackMoverLowEta/.gitignore

```diff
@@ -0,0 +1,5 @@
+BTagCalibrationStandalone.h
```

Comment:
Same as for high eta: do we need this directory at all (I hope we can replace it with a switch or options), and do we actually need this gitignore?